docs/Localization.md

```diff
@@ -0,0 +1,45 @@
+# Loop Localization
+
+## Background
+
+Loop (aka Firefox Hello) contains two parts:
+
+* A component that is built into, and shipped with Firefox desktop
+* A web page (also known as "Standalone") that is hosted on a web server.
+
+Releases to desktop and standalone happen every few weeks, however they are
+not always in sync.
+
+## Localizing Loop
+
+Loop localization is is via
+[Pontoon](https://pontoon.mozilla.org/projects/firefox-hello/). If you wish to
+help translate, please sign in there and suggest strings.
+
+## Testing your latest Localization strings
+
+Since releases only happen every few weeks, we have set up a development channel
+for localisers to test the latest code and check their strings.
+
+We have created a special add-on which gets updated about twice a week with the
+latest code and localized strings.
+
+*The add-on will override your installed Loop instance, and may cause any
+Loop data you previously had to be lost (if you had not signed in).*
+
+The add-on will create links to the development web pages. These also include
+the latest strings and are updated every half hour.
```

Comment:
Does "development web pages" actually mean "the development version of the standalone/link-clicker web app"?